electives/mocha-testing.md

```diff
@@ -240,7 +260,7 @@ describe("fizzBuzz", function(){
 
 ### Challenge
 
-No challenge yet!
+Do the Mocha and Chai projects have their own internal tests? Read them and see how they are used.
```

Comment:
Is part of the challenge to find the docs or codebase too? If not, please give them some links for where to start.

Reply:
Done